src/monodroid/monodroid.targets

```diff
@@ -24,6 +24,15 @@
     <GenerateMonoDroidIncludes SourceFiles="@(_EmbeddedBlobSource)" DestinationFiles="@(_EmbeddedBlobDestination)" />
   </Target>
 
+  <Target Name="RunStaticAnalysis"
```

Comment:
Should this target *always* be run, and the resulting outputs included in some build output?

Reply:
I don't think it should always run. We might want to run it as part of our CI, but otherwise it would be just wasted time for 99.9% of people.